hw/hw06/series.py

```diff
@@ -0,0 +1,80 @@
+# HW6 Mathmatical Series
+
+
+def fibonacci(n):
+    """Defining a function  called fibonacci to provide nth value in fibonacci
+    sequence."""
+    if n == 0:
+        return 0
+    # First value in fibonacci sequence is 0.
+    elif n == 1:
+        return 1
+    # Second value in fibonacci sequence is 1
+    else:
+        return fibonacci(n-1) + fibonacci(n-2)
```

Comment:
For readability, consider pep8: E226 whitespace, so: fibonacci(n - 1) + fibonacci(n - 2)